src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -192,9 +250,42 @@ async def _handle_unary_stream_rpc(object method_handler,
     )
 
     cdef tuple ops = (op,)
+    rpc_state.status_sent = True
     await execute_batch(rpc_state, ops, loop)
 
 
+async def _handle_exceptions(RPCState rpc_state, object rpc_coro, object loop):
+    try:
+        try:
+            await rpc_coro
+        except AbortError as e:
+            # Caught AbortError check if it is the same one
+            assert rpc_state.abort_exception is e, 'Abort error has been replaced!'
+            return
+        else:
+            # Check if the abort exception got suppressed
+            if rpc_state.abort_exception is not None:
+                _LOGGER.error(
+                    'Abort error unexpectedly suppressed: %s',
+                    traceback.format_exception(rpc_state.abort_exception)
+                )
+    except KeyboardInterrupt:
```

Comment:
Nit: These two can be a single `except` clause:

```python
except (KeyboardInterrupt, SystemExit):
  raise
```

Reply:
Done. (I thought it looks cooler to be separated, but in CPython code, they use the same catch clause for both of them.)